scripts/test-automation/README.txt

```diff
@@ -0,0 +1,67 @@
+Introduction - This is an end to end test where we test two Busybox containers, hosted on same or different hosts, ping each other. The connectivity between the two Busybox containers is provided by Alcor services and Alcor Control Agent.
```

Comment:
@kiran1048 Thanks for putting the instruction here. Could you make the README.txt in a markdown form README.md, just like our other README file https://github.com/futurewei-cloud/alcor/blob/master/README.md.

Let me know if you hit any issue, should be straightforward.

Reply:
Updated in correct format.